contributing.md

```diff
@@ -25,7 +25,7 @@ Pull Requests \(PRs\) are the main and exclusive way to contribute to the offici
 
 * Code Coverage does not decrease.
 * All the tests pass.
-* The code is formatted according to [![PEP8](https://img.shields.io/badge/code%20style-pep8-orange.svg)](https://www.python.org/dev/peps/pep-0008/).
+* The code is formatted according to [![PEP8](https://img.shields.io/badge/code%20style-pep8-orange.svg)](https://www.python.org/dev/peps/pep-0008/)
```

Comment:
But wait - that's not a typo, the sentence ends with a dot.

Reply:
@vmarkovtsev  usually badge doesn't ends with dot